blazor/sidebar/how-to/sidebar-in-the-dotnet8-application.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: Add Sidebar to MainLayout of the .NET 8 Blazor Web app | Syncfusion
+description: Learn here all about how to render the Sidebar component in the MainLayout page of the .NET 8 Blazor Web application and more.
+platform: Blazor
+control: Sidebar
+documentation: ug
+---
+
+<!-- markdownlint-disable MD009 -->
+
+# Render the Sidebar within the MainLayout of the .NET 8 Blazor Web app
+
+Integrate the Blazor Sidebar component into the `MainLayout.razor` page of the .NET 8 application. Next, include the `@rendermode InteractiveServer` directive in the `Routes.razor` page of the application.
+
+```cshtml
+
+@using Syncfusion.Blazor.Navigations
+@using Syncfusion.Blazor.Inputs
+<div class="page">
+    <div id="wrapper">
+        <div>
+            @*Initialize the Toolbar component*@
+            <SfToolbar>
+                <ToolbarEvents Clicked="@Toggle"></ToolbarEvents>
+                <ToolbarItems>
+                    <ToolbarItem PrefixIcon="e-tbar-menu-icon tb-icons" TooltipText="Menu"></ToolbarItem>
+                    <ToolbarItem>
+                        <Template>
+                            <div class="e-folder">
+                                <div class="e-folder-name">Navigation Pane</div>
+                            </div>
+                        </Template>
+                    </ToolbarItem>
+                </ToolbarItems>
+            </SfToolbar>
+        </div>
+        @*Initialize the Sidebar component*@
+        <SfSidebar HtmlAttributes="@HtmlAttribute" @ref="Sidebar" Width="290px" Target=".e-main-content" MediaQuery="(min-width:600px)" @bind-IsOpen="SidebarToggle">

```

Comment:
**HtmlAttributes** property is deprecated, so don't use that. use another way to achieve the requirement.

Reply:
Done.